lib/rex/proto/ssh/connection.rb

```diff
@@ -149,23 +150,6 @@ def close
   attr_reader :server, :context
 end
 
-##
-# Create a monitored relay between channel IOs and external FD-like objects
-##
-class ChannelRelay
```

Comment:
This seemed dead; And the `initialize_abstraction` call doesn't match the argument fo src/dst/threadname - so I've  nuked it to remove any confusion.